specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2024-04-01/mfe.json

```diff
@@ -19466,11 +19813,374 @@
+      "allOf": [
+        {
+          "$ref": "#/definitions/PackageInputPathBase"
+        }
+      ],
+      "properties": {
+        "resourceName": {
+          "description": "Input resource name.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "resourceVersion": {
+          "description": "Input resource version.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-ms-discriminator-value": "PathVersion",
+      "additionalProperties": false
+    },
+    "PackageInputType": {
+      "description": "Type of the inputs.",
+      "enum": [
+        "UriFile",
+        "UriFolder"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "PackageInputType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "UriFile"
+          },
+          {
+            "value": "UriFolder"
+          }
+        ]
+      },
+      "additionalProperties": false
+    },
+    "PackageRequest": {
+      "description": "Model package operation request properties.",
+      "required": [
+        "inferencingServer",
+        "targetEnvironmentId"
+      ],
+      "type": "object",
+      "properties": {
+        "baseEnvironmentSource": {
+          "description": "Base environment to start with.",
+          "$ref": "#/definitions/BaseEnvironmentSource",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "environmentVariables": {
+          "description": "Collection of environment variables.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "x-nullable": true
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "inferencingServer": {
+          "description": "[Required] Inferencing server configurations.",
+          "$ref": "#/definitions/InferencingServer",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "inputs": {
+          "description": "Collection of inputs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ModelPackageInput"
+          },
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true,
+          "x-ms-identifiers": []
+        },
+        "modelConfiguration": {
+          "description": "Model configuration including the mount mode.",
+          "$ref": "#/definitions/ModelConfiguration",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ],
+          "x-nullable": true
+        },
+        "properties": {
+          "description": "Property dictionary. Properties can be added, removed, and updated.",
+          "type": "object",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

ARM anti-pattern not allowed. Please schematize this object or model it as an array.